pagopa/io-app
# chore: [IOBP-574] Adds `success` flag to payment's Zendesk support payload

## Short description
This PR adds `success` flag to payment's Zendesk support payload

## List of changes proposed in this pull request
- Added `success: true` if outcome is `SUCCESS` (`0`)
- Added unit tests

## How to test
Within **Payments** playgrounds, start a new payment flow. Check that the payment's history is populated with the `success: true` value if outcome is success
